src/mgmt/rpc.h

```diff
@@ -124,7 +127,22 @@ CRT_RPC_DECLARE(mgmt_profile, DAOS_ISEQ_MGMT_PROFILE,
 CRT_RPC_DECLARE(mgmt_pool_get_svcranks, DAOS_ISEQ_MGMT_POOL_GET_SVCRANKS,
 		DAOS_OSEQ_MGMT_POOL_GET_SVCRANKS)
 
-#define DAOS_ISEQ_MGMT_TGT_CREATE /* input fields */		 \
+#define DAOS_ISEQ_MGMT_POOL_FIND /* input fields */		 \
+	((int32_t)		(pfi_bylabel)		CRT_VAR) \
+	((d_const_string_t)	(pfi_label)		CRT_VAR) \
+	((uuid_t)		(pfi_puuid)		CRT_VAR)
+
+#define DAOS_OSEQ_MGMT_POOL_FIND /* output fields */		 \
+	((uuid_t)		(pfo_puuid)		CRT_VAR) \
+	((d_rank_list_t)	(pfo_ranks)		CRT_PTR) \
+	((int32_t)		(pfo_rc)		CRT_VAR)
+
+CRT_RPC_DECLARE(mgmt_pool_find, DAOS_ISEQ_MGMT_POOL_FIND,
+		DAOS_OSEQ_MGMT_POOL_FIND)
+
+#define MGMT_POOL_FIND_DUMMY_LABEL "NO LABEL, FINDING BY UUID"
+
+		#define DAOS_ISEQ_MGMT_TGT_CREATE /* input fields */		 \
```

Comment:
(style)  line over 80 characters
(style)  Macros with complex values should be enclosed in parentheses